i confirmed the original issue; running the tests i pr'd to this branch on master gives us this error:

```

Runtime:       PHP 7.1.19 with Xdebug 2.5.5
Configuration: /Users/bc/tripal/sites/all/modules/tripal/phpunit.xml

E                                                                   1 / 1 (100%)

Time: 3.1 seconds, Memory: 58.00MB

There was 1 error:

1) Tests\ChadoQueryTest::test_filter_level
PDOException: SQLSTATE[42601]: Syntax error: 7 ERROR:  syntax error at or near ","
LINE 1: ...ock WHERE stock_id = '480' AND uniquename = 'LIKE', 'octopus...
                                                             ^
```

tests pass with @guignonv 's PR so thank you.

Can you please just  address https://github.com/tripal/tripal/pull/827#discussion_r250619969 ?  Then we should be good to merge.

--- a/tripal_chado/api/tripal_chado.query.api.inc
+++ b/tripal_chado/api/tripal_chado.query.api.inc
@@ -1368,7 +1368,7 @@ function chado_select_record($table, $columns, $values, $options = NULL) {
       // CASE 1a: If there is only one element in the array, treat it the same
       // as a non-array value.
       if (count($value) == 1 AND is_int(key($value))
-          AND !(isset($value[0]['op']) && isset($value[0]['op']))) {
+          AND !(isset($value[0]['op']) && isset($value[0]['data']))) {
 
         $value = array_pop($value);
         $op = '=';

